src/dashboard/DashboardComponent.vue

```diff
@@ -3,23 +3,54 @@ div.container
   div.row
     div.col.mt-4
       h1 Thankful
+    div.col
+      b-alert(show).mt-2.mb-0.p-2
+        h5 We're in alpha!
+        div
+          | Things are not all they could be and will break, but we're working on it.
+  div.row
+    div.col
       hr
 
   div.row
-    div(v-if="creators.length === 0")
-      | No creators to show
     div.col-md-6
       h3 Creators
+      div(v-if="creators.length === 0")
+        | No creators to show
       creator-card(v-for="creator in creators",
                    v-bind:creator="creator",
                    v-bind:key="creator.url",
-                   @allocatedFunds="creator.allocatedFunds = $event"
-                   @address="creator.address = $event"
+                   @allocatedFunds="creator.allocatedFunds = $event; creator.save();"
+                   @address="creator.address = $event; creator.save();"
                    )
 
       b-button(variant="success", size="lg", v-on:click="donateAll()")
         | Donate {{ totalAllocated }}$
 
+      hr
+
+      h3 Unattributed Activity
```

Comment:
Put inside a container with a scrollbar or something, this makes the page very long otherwise.

Reply:
Done